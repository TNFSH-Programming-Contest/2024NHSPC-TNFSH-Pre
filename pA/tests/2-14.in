858553009693487904 979410234912194567 921534672919249175
29168691211987962 345922582609961123 725638006739982677
