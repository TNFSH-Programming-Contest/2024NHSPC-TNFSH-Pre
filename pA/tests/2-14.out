1100729280561931762
